pom.xml

```diff
@@ -39,7 +39,7 @@
     <properties>
         <!-- Primary dependencies - maintained manually -->
         <camel.major.minor>3.11</camel.major.minor> <!-- run after each change: cd docs && mvnd validate -->
-        <camel.version>${camel.major.minor}.1</camel.version>
+        <camel.version>${camel.major.minor}.5</camel.version>
```

Comment:
You'll need to bump the parent version too.

Maybe we should introduce a CI check to make sure things are in sync?

Reply:
I think I did it some where already, but apparently not in this branch. Thanks for the catch!